tests/ui/inefficient_to_string.rs

```diff
@@ -0,0 +1,31 @@
+#![deny(clippy::inefficient_to_string)]
```

Comment:
Can you add `// run-rustfix` to make this a [`rustfix` test](https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md#rustfix-tests)?